Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:21-alpine AS build
+
+WORKDIR /app
+RUN mkdir node_modules && chown -R node:node .
+USER node
+
+ADD --chown=node:node package*.json ./
+RUN npm install
```

Comment:
Should we be utilising the lock file with `npm ci`? 🤔

Reply:
Yes, I'll change it.